Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -691,6 +694,134 @@ public class CameraManager {
+     - Returns `CameraAnimator`: A class that represents an animator with the provided configuration.
+     */
+    func makeCameraAnimator(duration: TimeInterval,
+                            curve: UIView.AnimationCurve,
+                            animationOwner: AnimationOwnerProtocol = AnimationOwner.unspecified,
+                            animations: (() -> Void)? = nil) -> CameraAnimator {
+        let propertyAnimator = UIViewPropertyAnimator(duration: duration, curve: curve, animations: animations)
+        let cameraAnimator = CameraAnimator(delegate: self, propertyAnimator: propertyAnimator, owner: animationOwner)
+        cameraAnimators.add(cameraAnimator)
+        return cameraAnimator
+    }
+
+    /**
+     Convenience to create a `CameraAnimation` and will add it to a List of `CameraAnimators` to track the lifecycle of that animation
+
+     - Parameter duration: The duration of the animation, in seconds.
+     - Parameter controlPoint1: The first control point for the cubic Bézier timing curve.
+     - Parameter controlPoint2: The second control point for the cubic Bézier timing curve.
+     - Parameter animationOwner: The `id` of what owns this animation.
+     - Parameter animations: The block containing the animations. This block has no return value and takes no parameters. Use this block to modify any animatable view properties. When you start the animations, those properties are animated from their current values to the new values using the specified animation parameters.
+     - Returns `CameraAnimator`: A class that represents an animator with the provided configuration.
+     */
+    func makeCameraAnimator(duration: TimeInterval,
+                            controlPoint1 point1: CGPoint,
+                            controlPoint2 point2: CGPoint,
+                            animationOwner: AnimationOwnerProtocol = AnimationOwner.unspecified,
+                            animations: (() -> Void)? = nil) -> CameraAnimator {
+        let propertyAnimator = UIViewPropertyAnimator(duration: duration, controlPoint1: point1, controlPoint2: point2, animations: animations)
+        let cameraAnimator = CameraAnimator(delegate: self, propertyAnimator: propertyAnimator, owner: animationOwner)
+        cameraAnimators.add(cameraAnimator)
+        return cameraAnimator
+    }
+
+    /**
+     Convenience to create a `CameraAnimation` and will add it to a List of `CameraAnimators` to track the lifecycle of that animation
+
+     - Parameter duration: The duration of the animation, in seconds.
+     - Parameter timingParameters: The type of `AnimationCurve` for the animation.
+     - Parameter animationOwner: The `id` of what owns this animation.
```

Comment:
What do you mean by `id`? This is just a conformer to the `AnimationOwnerProtocol` right?

Reply:
correct, but the protocol when used is representing a way to ID the animator

Reply:
The `id` isn't all the animationOwner could have -- it just happens to have that at this time. That will change in the future.